src/module/item/mixins/item-activation.js

```diff
@@ -29,10 +43,22 @@ export const ItemActivationMixin = (superclass) => class extends superclass {
     }
 
     isActive() {
-        const itemData = this.system;
-        return !!(itemData.isActive);
+        return !!this.system.isActive;
     }
 
+    get timeRemainingStr() {
+        return this.system.activationEvent?.status
+            ?? this.timedEffect?.activeDuration?.remaining?.string
+            ?? game.i18n.localize("SFRPG.ActorSheet.Inventory.Item.Deactivate");
+    }
+
+    /**
+     * Activate or deactivate this item.
+     * @param {!boolean} active True to activate, false to deactivate.
+     * @param {?ActivationEvent} activationEvent Optional activation event to be used in place of an auto-computed one.
```

Comment:
Not sure if this parameter is still used? `setActive` only has the `active` parameter